doc/pilight-daemon.1

```diff
@@ -0,0 +1,88 @@
+.TH pilight-daemon 1 "10 July 2017" "7.0-dev" "pilight 7.0-dev"
+.SH NAME
+pilight-daemon \- The pilight daemon
+.SH SYNOPSIS
+.B pilight-daemon
+.br
+.B pilight-daemon [\fIOPTION\fR]...
+.br
+.B pilight-daemon [\fI--server SERVER\fR] [\fI --port PORT\fR] [\fIOPTION\fR]...
+.SH DESCRIPTION
+.B pilight-daemon
+is the main software of the pilight project. It is responsible for handling and
+interpreting received signals, as well as generating signals to communicate
+with and control various devices.
+.PP
+A configuration file is required. By default, /etc/pilight/config.json is used.
+Currently, configuration files have to be created in the JSON format, although
+support for other types of configuration files may be added in the future. 
+.PP
+Consult the pilight wiki <https://wiki.pilight.org/> and the manual
+<https://manual.pilight.org/> for details on how to configure and use pilight. 
+.PP
+By default, the pilight daemon also hosts a webgui which allows controlling 
+devices through a clickable interface in a webbrowser.
+The pilight daemon also runs a socket server and provides an API for 
```

Comment:
I would say `REST` API and change `from external programs` =>`to interface with`.